src/prefect/server/models/block_registration.py

```diff
@@ -174,7 +174,7 @@ async def _register_collection_blocks(session: AsyncSession) -> None:
 
     # due to schema reference dependencies, we need to register all block types first
     # and then register all block schemas
-    block_schemas: dict[str, dict] = {}
+    block_schemas: Dict[str, Dict[str, Any]] = {}
```

Comment:
```suggestion
    block_schemas: dict[str, dict[str, Any]] = {}
```

Reply:
That's interesting, I was getting errors about these from pre-commit, hmm